docs/user-guide.md

```diff
@@ -312,76 +414,119 @@ pipelines:
         provider: codebuild
         role: packer
         size: medium
-    triggers: # What should trigger this pipeline, and what should be triggered when it completes
+    # What should trigger this pipeline, and what should be triggered
+    # when it completes
+    triggers:
       triggered_by:
         code_artifact:
           repository: my_test_repository
 ```
 
-In the above example, the *ami-builder* pipeline is triggered when a new package version is published to the *my_test_repository* repository in CodeArtifact.
+In the above example, the *ami-builder* pipeline is triggered when a new package
+version is published to the *my_test_repository* repository in CodeArtifact.
 
 ### Additional Deployment Maps
 
 You can also create additional deployment map files.
 These can live in a folder in the pipelines repository called *deployment_maps*.
-These are entirely optional, but can help split up complex environments with many pipelines.
+These are entirely optional, but can help split up complex environments
+with many pipelines.
 
-For example, you might have a map used for infrastructure type pipelines and one used for deploying applications.
-These additional deployment map files can have any name, as long as they end with *.yml*.
+For example, you might have a map used for infrastructure type pipelines and
+one used for deploying applications. These additional deployment map files can
+have any name, as long as they end
+with `.yml`.
 
 Taking it a step further, you can create a map per service.
-So you can organize these deployment map files inside your preferred directory structure
-For example, the `aws-deployment-framework-pipelines` repo could look like this:
+So you can organize these deployment map files inside your preferred directory
+structure. For example, the `aws-deployment-framework-pipelines` repo could look
+like this:
 
-```
+```txt
 deployment_maps/
-  security/
-    amazon-guardduty.yml
-    aws-config.yml
-  product-one/
-    roles/
-      some-role-pipeline.yml
-    infrastructure/
-      some-infra-pipeline.yml
+|-- security/
+|   |-- amazon-guardduty.yml
+|   |-- aws-config.yml
+|-- product-one/
+|   roles/
+|   |-- some-role-pipeline.yml
+|   infrastructure/
+|   |-- some-infra-pipeline.yml
```

Comment:
To make it use the same structure as used in the `admin-guide.md` file with lists:

```suggestion
├───security/
|   ├───amazon-guardduty.yml
|   └───aws-config.yml
└───product-one/
    ├───roles/
    |   └───some-role-pipeline.yml
    └───infrastructure/
        └───some-infra-pipeline.yml
```